fastestimator/trace/io/model_eval.py

```diff
@@ -0,0 +1,451 @@
+                itemize.add_item(
+                    escape_latex("Execution time: {:.2f} seconds".format(self.json_summary['execution_time(s)'])))
+
+            with self.doc.create(Table(position='H')) as table:
+                table.append(NoEscape(r'\refstepcounter{table}'))
+                self._document_summary_table(pass_num=len(instance_pass_tests) + len(aggregate_pass_tests),
+                                             fail_num=len(instance_fail_tests) + len(aggregate_fail_tests))
+
+        if instance_fail_tests or aggregate_fail_tests:
+            with self.doc.create(Section("Failed Tests")):
+                if len(aggregate_fail_tests) > 0:
+                    with self.doc.create(Subsection("Failed Aggregate Tests")):
+                        with self.doc.create(Table(position='H')) as table:
+                            table.append(NoEscape(r'\refstepcounter{table}'))
+                            self._document_aggregate_table(tests=aggregate_fail_tests)
+                if len(instance_fail_tests) > 0:
+                    with self.doc.create(Subsection("Failed Per-Instance Tests")):
+                        with self.doc.create(Table(position='H')) as table:
+                            table.append(NoEscape(r'\refstepcounter{table}'))
+                            self._document_instance_table(tests=instance_fail_tests, with_ID=self.data_id)
+
+        if instance_pass_tests or aggregate_pass_tests:
+            with self.doc.create(Section("Passed Tests")):
+                if aggregate_pass_tests:
+                    with self.doc.create(Subsection("Passed Aggregate Tests")):
+                        with self.doc.create(Table(position='H')) as table:
+                            table.append(NoEscape(r'\refstepcounter{table}'))
+                            self._document_aggregate_table(tests=aggregate_pass_tests)
+                if instance_pass_tests:
+                    with self.doc.create(Subsection("Passed Per-Instance Tests")):
+                        with self.doc.create(Table(position='H')) as table:
+                            table.append(NoEscape(r'\refstepcounter{table}'))
+                            self._document_instance_table(tests=instance_pass_tests, with_ID=self.data_id)
+
+    def _document_summary_table(self, pass_num: int, fail_num: int) -> None:
+        """Document a summary table.
+
+        Args:
+            pass_num: Total number of passed tests.
+            fail_num: Total number of failed tests.
+        """
+        with self.doc.create(Tabularx('|Y|Y|Y|', booktabs=True)) as tabular:
+            package = Package('seqsplit')
+            if package not in tabular.packages:
+                tabular.packages.append(package)
+
+            # add table heading
+            tabular.add_row(("Total test #", "Total pass #", "Total fail #"), strict=False)
+            tabular.add_hline()
+
+            tabular.add_row((pass_num + fail_num, pass_num, fail_num), strict=False)
```

Comment:
I forget, what is strict=False for?

Reply:
With strict = True, add_row method will check if the number of the cells of the incoming row matches with the number of table column. Because we use redefined column type Y, the tabular cannot count the column correctly, and therefore we disable this check.

Reply:
interesting. is that captured in a comment anywhere?